lib/taskana-core/src/main/java/pro/taskana/TaskService.java

```diff
@@ -56,6 +56,38 @@ Task claim(String taskId)
     Task claim(String taskId, boolean forceClaim)
         throws TaskNotFoundException, InvalidStateException, InvalidOwnerException;
 
+    /**
+     * Unclaim a existing Task which was claimed and owned by you before.
+     *
+     * @param taskId
+     *            id of the task which should be unclaimed.
+     * @return updated unclaimed task
+     * @throws TaskNotFoundException
+     *             if the task can´t be found or does not exist
+     * @throws InvalidStateException
+     *             when the task is already completed.
+     * @throws InvalidOwnerException
+     *             when the unclaim is not forced and user is diffrent.
+     */
+    Task cancelClaim(String taskId) throws TaskNotFoundException, InvalidStateException, InvalidOwnerException;
+
+    /**
+     * Unclaim a existing Task which was claimed and owned by you before. Also there can be enabled a force flag for
+     * admins.
+     *
+     * @param taskId
+     *            id of the task which should be unclaimed.
+     * @return updated unclaimed task
```

Comment:
Javadoc for forceUnclaim missing. In addition, the name forceCancel or simply force for this parameter would be better